.github/pull_request_template.md

```diff
@@ -0,0 +1,31 @@
+Tag name (required for release branches):
+Originator(s):
+
+Summary (include the keyword ['closes', 'fixes', 'resolves'] and issue number):
+
+Describe any changes made to build system:
+
+Describe any changes made to the namelist:
+
+List any changes to the defaults for the input datasets (e.g. boundary datasets):
+
+List all files eliminated and why:
+
+List all files added and what they do:
+
+List all existing files that have been modified, and describe the changes: 
+(Helpful git command: `git diff --name-status development...<your_branch_name>`)
+
+If there are new failures (compare to the `test/existing-test-failures.txt` file),
```

Comment:
Maybe change `compare `=> `compared` ?  I can see it either way

Reply:
done! i agree that "compared" is a bit better